src/components/account/edit/sections/LecturerInformationSection.vue

```diff
@@ -3,7 +3,7 @@
         <div class="lg:flex">
             <div class="flex flex-col w-full mb-4 mr-12 lg:w-1/3 lg:block">
                 <label class="block mb-2 text-lg font-medium text-gray-700">Lecturer Information</label>
-                <label class="block text-gray-600"> Information Specifically for a Lecturer </label>
+                <label class="block text-gray-600">Manage relevant details of the new lecturer account.</label>
```

Comment:
```suggestion
                <label class="block text-gray-600">Manage relevant details of the lecturer account.</label>
```
The lecturer account does not have to be new, as you see the same label in editmode